src/utils/duration-to-human.ts

```diff
@@ -0,0 +1,29 @@
+import { Duration } from 'luxon';
+
+function pluralize(str: string, val: number) {
+  return val === 1 ? str : str + 's';
+}
+
+export default function durationToHuman(duration: Duration): string {
```

Comment:
I would appreciate a comment here with a sample output 😄

Reply:
sure thing! Fwiw I *did* add a bunch of tests. But it's nice to see the sample inline too :)